storage/innobase/xtrabackup/doc/source/release-notes/2.3/2.3.4.rst

```diff
@@ -0,0 +1,23 @@
+============================
+ |Percona XtraBackup| 2.3.4
+============================
+
+Percona is glad to announce the release of |Percona XtraBackup| 2.3.4 on March 17th 2015. Downloads are available from our download site `here <http://www.percona.com/downloads/XtraBackup/xtrabackup-2.3.4/>`_. Downloads are available from our `download site <http://www.percona.com/downloads/XtraBackup/LATEST/>`_ and from :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+This release is the current GA (Generally Available) stable release in the 2.3 series.
+
+Bugs fixed
+----------
+
+ |Percona XtraBackup| didn't respect :option:`innodb_log_file_size` variable stored in :file:`backup-my.cnf`. Bug fixed :bug:`1527081`.
+
+ If server would run out of space while backups were taken with :option:`innobackupex --rsync` option backup process would fail but |innobackupex| would still complete with ``completed OK!`` message. Bug fixed :bug:`1537256`.
+
+ |Percona XtraBackup| would not log master coordinates while performing backup on *MariaDB* 10. Bug fixed :bug:`1404484`.
```

Comment:
didn't store gtid_slave_pos in xtrabackup_slave_info for mariadb 10 with GTID. It was only logged to stderr